macros/edr/tests/test_utils/backfill_result_rows.sql

```diff
@@ -0,0 +1,35 @@
+{% macro backfill_result_rows() %}
```

Comment:
Some thoughts / questions about it:

- I assume we should tell in the release notes for people to execute `dbt run -s elementary` once, to create the new table?
- I'm not sure if we have performance concerns, though since it's limited by days_back it's probably fine.
- WDYT about concurrent executions of this - are we worried about it?

Not sure we should change anything, just wonder what you think about it.

Reply:
1. Yes, they'll need to run the new table. Do you think we should add a check at the `on-run-start` that the table exists? I don't want to break the pipeline but maybe it makes sense if they upgrade. Otherwise test results' will just be list.
2. Yes, that's mostly what I figured as well. Do you have an idea on how to optimize it further?
3. I don't think so because if there's a concurrent run, the materialization of the model itself would crash and therefore there wouldn't be a call to the `post-hook` (this macro). I'm validating that the table doesn't exist yet (`not is_incremental()`).

Reply:
1. I think as long as it's clear what they need to do it may be fine (unless you think it's easy to do the on-run-start change).
2. Not sure it's needed, let's learn like Or says :)
3. Yup, makes sense to me.